src/common/client.py

```diff
@@ -18,9 +20,13 @@ class EtcdClient:
 
     def __init__(
         self,
+        username: str,
```

Comment:
There are cases where we could omit it (e.g. `endpoints status`). Still I'd like to always instantiate the client the same way, just to be consistent. Otherwise we might have to adjust the instantiation of the client  alot in the future when we mix auth-requiring with non-auth commands in the same instance.